extension/commands/debugCommandListCommand.ts

```diff
@@ -66,9 +67,9 @@ export async function debugCommandList(commandName?: string): Promise<void> {
 
         if (!FabricConnectionManager.instance().getConnection()) {
             // Connect to local_fabric gateway before submitting/evaluating transaction
-            const runtimeGateways: Array<FabricGatewayRegistryEntry> = await FabricRuntimeManager.instance().getGatewayRegistryEntries();
```

Comment:
Did we definitely want to delete the `getGatewayRegistryEntries` function? We won't want support for multiple managed runtimes in future?

Reply:
even if we have multiple managed runtimes we don't need this as all the registry entries will be in the gateways folder